tests/unit-tests/config/__snapshots__/semi-false.test.ts.snap

```diff
@@ -81,6 +81,24 @@ class MyComponent extends Component {
 "
 `;
 
+exports[`config > semi: false > it formats ../cases/gjs/component-class-with-required-semi-before-template.gjs 1`] = `
+"import Component from "@glimmer/component"
+
+/** It's a component */
+class MyComponent extends Component {
+  get rows() {
+    return 5
+  }
+
+  historyBack = () => history.back();
```

Comment:
I manually edited this snapshot to correct it (I put the semicolon in here), so that the test would fail as the bug has not been fixed as of writing

Reply:
Perfect.